### PR TITLE
Upgrade to docusaurus v2.2.0

### PR DESCRIPTION
**Pull request by oredavids**
#### Sanity Check
- [x] Docusaurus now requires node v16.14.0; do we need to make infra changes for stage & prod?

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -243,11 +243,6 @@ const config = {
         defaultMode: "light",
         disableSwitch: false,
         respectPrefersColorScheme: false,
-        /* we clear these icons out and use CSS to swap in our own images instead */
-        switchConfig: {
```
Comment (eak12913) [inline docusaurus.config.js:247]: Does this still work if you remove this config? Do we have to make this change somewhere else?

File: src/theme/Toggle/index.js [hunk 2]
```diff
@@ -1,77 +0,0 @@
-/**
```
Comment (eak12913) [inline src/theme/Toggle/index.js:1]: Were these not being customized in spite of having been swizzled?

**Review by eak12913 — COMMENTED** [review]
>Docusaurus now requires node v16.14.0; do we need to make infra changes for stage & prod?

I don't think we will need to make changes beyond what you did in CircleCI config. There's no `node` server running in prod. It's just generating a bunch of HTML files that are being uploaded into an S3 bucket.

File: docusaurus.config.js (hunk 1 above)
Comment (oredavids) [reply docusaurus.config.js:247]: This worked hand-in-hand with the swizzled Toggle Component. Now that we are no longer swizzling it; the config is not required.

File: src/theme/Toggle/index.js (hunk 2 above)
Comment (oredavids) [reply src/theme/Toggle/index.js:1]: The component was swizzled solely to customize the appearance via CSS. See from line 107 of the also deleted stylesheet [src/theme/Toggle/styles.module.css](https://github.com/gruntwork-io/docs/blob/f32a0783814c57feab50ea4746eab50a034ea0c2/src/theme/Toggle/styles.module.css#L107).

I showed @ebeneliason the new toggle in this version of Docusaurus and we agreed that there is no need to swizzle anymore as the new appearance is in line with what we implemented.

**Comment by oredavids** [comment]
> I don't think we will need to make changes beyond what you did in CircleCI config. There's no node server running in prod. It's just generating a bunch of HTML files that are being uploaded into an S3 bucket.

Ah, yes! Thank you.

**Review by eak12913 — APPROVED** [review]
LGTM!

**Pull request merged**